fundamentals/arrays/lesson_notes/lesson_04.md

```diff
@@ -0,0 +1,115 @@
+# Arrays, Destructuring & Rest
```

Comment:
I would split multidimensional arrays into its own section and expand it a little bit. 

I think this entire section could be `bonus` / skip if out of time or students are feeling overwhelmed. It can always be revisited and given as reading for hw/ bonus hw problems.

If torn between the two, I prefer showing multidimensional arrays since it is more practice accessing items in an array, practicing a more familiar syntax.

I worry that destructing is just too much information and would put it toward the end of the module/later

Reply:
I don't disagree with you. Right now the current Readme has this content in it so I created a lesson to accompany it, but I'd really rather move content like this to a few weeks after students have been working with JavaScript.

I think this is a discussion on what's the best change(s) to make for 8.0. That is, if there's content we don't think it's important to cover, do we make more substantive modifications to the actually curriculum or just omit talking about it in lessons...?